### ISSUE TITLE
Carousel Alignment Bug on iPhone X

### ISSUE THREAD

**Issue by brittanygay** [revelrylabs/bn-mobile-react#77]
The Slider Alignment on iPhone X is off, and using flexbox around the slider is doing some wonky things. 

We need to look deeper in the docs here to resolve the issue: https://github.com/archriss/react-native-snap-carousel

Screenshot:
<img width="336" alt="screen shot 2018-08-24 at 10 19 36 am" src="https://user-images.githubusercontent.com/14582709/44592787-3c5dbd80-a787-11e8-83cc-a753fa326173.png">

**Comment by iamjoshfrank** [comment]
@brittanygay - renaming this to "carousel" since that's what the plugin is called (and that's what the thing actually is).

i kept getting confused when i heard "slider." i was expecting to see a draggable, sliding input not an image carousel. :wink:

**Comment by brittanygay** [comment]
The Carousel alignment bug is now fixed, more info and screenshots here: https://github.com/revelrylabs/bn-mobile-react/pull/80

Assigning to @daybreaker for QA. 

**Note to @daybreaker:**
We still need to test on Android. I checked on iPhone X and other devices but since i don't have access to an Android today I wasn't able to check. I'll be in the office tomorrow so I should be able to help QA on Androids then!
